test/test_device.py

```diff
@@ -130,11 +129,12 @@ def create(*args, **kwargs):
         return c
 
 class DeviceTestCase(unittest.TestCase):
-    def __init__(self, bitcoind, emulator=None, interface='library', methodName='runTest'):
+    def __init__(self, bitcoind, emulator=None, interface='library', methodName='runTest', supports_legacy=True):
```

Comment:
`supports_legacy` should be a property of the emulator, not an argument to the test case. See `supports_taproot`.

Reply:
Okay, we can use this convention. I'll updating that shortly, but then I will also add the relevant field to the other emulators.